CondTools/SiPixel/plugins/SiPixelGenErrorDBObjectReader.h

```diff
@@ -33,7 +36,7 @@ class SiPixelGenErrorDBObjectReader : public edm::EDAnalyzer {
   bool theDetailedGenErrorDBErrorOutput;
   bool theFullGenErrorDBOutput;
   //bool testGlobalTag;
```

Comment:
I see no hint of these variables, even commented out, in the implementation code: they can probably be safely removed also here.